setup.py

```diff
@@ -78,7 +78,7 @@ def get_git_sha() -> str:
         "colorama",
         "croniter>=0.3.28",
         "cron-descriptor",
-        "cryptography>=39.0.1, <40",
+        "cryptography>=41.0.0",
```

Comment:
Should we reintroduce the lower bound (which could be increased) otherwise in theory we could be installing a version older than `39.0.1`.